src/sql/src/plan/query.rs

```diff
@@ -3496,7 +3493,20 @@ where
         .order_by
         .into_iter()
         .enumerate()
-        .map(|(i, ColumnOrder { column: _, desc })| ColumnOrder { column: i, desc })
+        .map(
+            |(
+                i,
+                ColumnOrder {
+                    column: _,
+                    desc,
+                    nulls_last,
+                },
+            )| ColumnOrder {
+                column: i,
+                desc,
+                nulls_last,
+            },
+        )
```

Comment:
Nit: This might be a better fit for rustfmt's layouting:
```rust
.map(|(i, order)| ColumnOrder {
    column: i,
    desc: oder.desc,
    nulls_last: order.nulls_last,
})
```

... or with less boilerplate:
```rust
.map(|(i, order)| ColumnOrder {
    column: i,
    ..order
})
```

Reply:
done

Reply:
This is really nice, btw. :)